px4-workspace/px4-msgs-sys/src/orb.rs

```diff
@@ -0,0 +1,156 @@
+pub type orb_id_size_t = u16;
+#[doc = " Object metadata."]
+#[repr(C)]
+#[derive(Debug, Copy, Clone)]
+pub struct orb_metadata {
+    #[doc = "< unique object name"]
+    pub o_name: *const ::core::ffi::c_char,
+    #[doc = "< object size"]
+    pub o_size: u16,
+    #[doc = "< object size w/o padding at the end (for logger)"]
+    pub o_size_no_padding: u16,
+    #[doc = "< Hash over all fields for message compatibility checks"]
+    pub message_hash: u32,
+    #[doc = "< ORB_ID enum"]
+    pub o_id: orb_id_size_t,
+    #[doc = "< queue size"]
+    pub o_queue: u8,
+}
+
+impl PartialEq for orb_metadata {
+    fn eq(&self, other: &Self) -> bool {
+        self.o_id == other.o_id
+    }
+}
+
+// #[allow(clippy::unnecessary_operation, clippy::identity_op)]
+// const _: () = {
+//     ["Size of orb_metadata"][::core::mem::size_of::<orb_metadata>() - 16usize];
+//     ["Alignment of orb_metadata"][::core::mem::align_of::<orb_metadata>() - 4usize];
+//     ["Offset of field: orb_metadata::o_name"]
+//         [::core::mem::offset_of!(orb_metadata, o_name) - 0usize];
+//     ["Offset of field: orb_metadata::o_size"]
+//         [::core::mem::offset_of!(orb_metadata, o_size) - 4usize];
+//     ["Offset of field: orb_metadata::o_size_no_padding"]
+//         [::core::mem::offset_of!(orb_metadata, o_size_no_padding) - 6usize];
+//     ["Offset of field: orb_metadata::message_hash"]
+//         [::core::mem::offset_of!(orb_metadata, message_hash) - 8usize];
+//     ["Offset of field: orb_metadata::o_id"][::core::mem::offset_of!(orb_metadata, o_id) - 12usize];
+//     ["Offset of field: orb_metadata::o_queue"]
+//         [::core::mem::offset_of!(orb_metadata, o_queue) - 14usize];
+// };
```

Comment:
This was an alignment test generated by bindgen that was failing when you switched between targets. We can probably just remove it